### PR TITLE
Solver per branch

### PR DESCRIPTION
**Pull request by norhh**

### CODE REVIEW

File: mythril/laser/ethereum/instructions.py [hunk 1]
```diff
@@ -1056,6 +1056,7 @@ def jumpi_(self, global_state):
                 states.append(new_state)
             else:
                 logging.debug("Pruned unreachable states.")
+        del global_state
```
Comment (JoranHonig) [inline mythril/laser/ethereum/instructions.py]: won't this be automatically be garbage collected?

Comment (norhh) [reply mythril/laser/ethereum/instructions.py]: I think so but I am not totally sure, I added del everywhere as a desperate attempt to reduce memory and this got left off.

File: mythril/analysis/modules/dependence_on_predictable_vars.py [hunk 2]
```diff
@@ -48,7 +48,7 @@ def execute(statespace):
 
         found = []
         for var in vars:
-            for constraint in call.node.constraints + [call.to]:
+            for constraint in call.node.constraints[:] + [call.to]:
```
Comment (JoranHonig) [inline mythril/analysis/modules/dependence_on_predictable_vars.py]: Is the [:] necessary here? (same for ln 78)

**Comment by JoranHonig** [comment]
@norhh maybe :innocent:

**Pull request merged**